src/css/profile/wearable/changeable/theme-circle/listview.less

```diff
@@ -53,36 +53,14 @@
 		align-items: center;
 
 		&.ui-listview-divider, &.ui-li-group-index {
-			height: 33 * @unit_base;
-			min-height: 33 * @unit_base;
-			line-height: 31 * @unit_base;
-			background-color: transparent;
-			color: @color_list_divider_text;
-			display: flex;
-			flex-direction: row;
+			height: 32 * @px_base;
```

Comment:
do we need to define both height and min-height?

Reply:
li for ui-listview has defined both min-height and max-height (see above). Here we need to overwrite them. Changed to max-height to have correct size.